database/nakadi/10_data/04_tables/02_subscription.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE IF NOT EXISTS zn_data.subscription (
+  s_id varchar(255) NOT NULL PRIMARY KEY CHECK (s_id <> ''),
```

Comment:
Are subscription IDs not UUID? 

If you are going paranoid about the text is, it would make sence to also trim the value in the check.